libraries/AP_Compass/AP_Compass_QMC5883P.cpp

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (C) 2016  Emlid Ltd. All rights reserved.
```

Comment:
why copyright emlid? better to remove the line completely and rely on git history